page-optimize.php

```diff
@@ -258,7 +264,8 @@ function page_optimize_init() {
 	}
 
 	// Schedule cache cleanup on init
-	if( ! wp_next_scheduled( PAGE_OPTIMIZE_CRON_CACHE_CLEANUP_JOB ) ) {
+	$using_cache = defined( 'PAGE_OPTIMIZE_CACHE_DIR' ) && ! empty( PAGE_OPTIMIZE_CACHE_DIR );
+	if( $using_cache && ! wp_next_scheduled( PAGE_OPTIMIZE_CRON_CACHE_CLEANUP_JOB ) ) {
```

Comment:
It doesn't matter here because AFAICT WordPress doesn't let you schedule a cron event with a timestamp of zero, but I think it is a good habit to be as specific as possible and compare directly against `false` since it will be truthy if the function returns integer zero.

`false !== wp_next_scheduled( PAGE_OPTIMIZE_CRON_CACHE_CLEANUP_JOB )`

I don't think it needs to be changed but wanted to comment since, in my work, it's easier to be explicit all the time than to notice and adjust when it does matter.

Reply:
Will update tomorrow. Prolly needs to be a `false ===` check.